homework-submission/myapp/src/components/checkBox.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { observer, inject } from "mobx-react";
+
+@inject("todoStore")
+@observer
+class CheckBox extends React.Component {
+  
+  render() {
+    const { todoStore, done, id } = this.props;
+
```

Comment:
You didn't pass `done, id` while you are using them here
```suggestion
   console.log(done, id);
```